internal/gencli/testdata/manage-todos.want

```diff
@@ -0,0 +1,104 @@
+// Code generated. DO NOT EDIT.
+
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/spf13/cobra"
+
+	todopb "github.com/googleapis/todo/generated"
+)
+
+var ManageTodosInput todopb.Todo
```

Comment:
Let's put this in the function, I don't think it's needed in the package scope.

Reply:
These need to be globally scoped so that both the `pflag` configuration in the following `init()` function and the command execution function given to `RunE` can use them. 

I could make them unexported if that helps?

Reply:
`ManageTodosInput` only appears in one function though. Is this the case where it's needed in `init()` some of the times?

Reply:
Ahh in this case it is a client streaming method, so it is only used in the command execution. Typically the variable is used elsewhere (in `init()`), but in this case it is not.

I can account for this case and only declare the global variable in non-client streaming cases.